### PR TITLE
Added Quantity to book class and fixed report listings

### PR DESCRIPTION
**Pull request by scrable**
List Inventory wasn't sorting properly before. Fixed the sorting logic
to correctly display the sorted database.

### CODE REVIEW

File: CIS22B_FinalProject/CIS22B_FinalProject/Report.cpp [hunk 1]
```diff
@@ -24,122 +24,174 @@ Report::Report(BookDatabase* dtbs)
...
 //books that have been in the inventory longest will be listed first.
 void Report::listAge()
 {
+	system("CLS"); //clear screen of other modules and text
 	cout << "Listing inventory by date added...\n\n";
-	//write newly sorted book database to file
+
+	Book* books = database->getBooks();
+	int size = database->getSize();
+
```
Comment (scrable) [inline CIS22B_FinalProject/CIS22B_FinalProject/Report.cpp:126]: listAge() is incomplete, does not work properly yet

**Comment by ahouts** [comment]
two books can actually have the same isbn. In fact, chances are there will be many books with the same isbn. The only thing that should be unique to any given book is its identifier, and the user never has any control over the identifier of any book.

**Comment by ahouts** [comment]
Also you pushed your database files. I think you can edit your push if you go to the files changed portion of the commit. If you change both of those I will accept this pull request

**Comment by ahouts** [comment]
To edit it just make some commits on github desktop that make those changes press the sync button in the top right. That will update your pull request will all of the changes you have made since making the pull request.

**Comment by ahouts** [comment]
looks good now :)

**Pull request merged**